drivers/thermal/Kconfig

```diff
@@ -120,13 +120,12 @@ config THERMAL_QPNP_ADC_TM
 
 config THERMAL_MONITOR_SIMPLE
 	bool "Simple solution to monitor CPU temp and limit CPU Frequency"
-	depends on THERMAL_TSENS8960 || THERMAL_TSENS8974
 	depends on CPU_FREQ_MSM
 	depends on !THERMAL_MONITOR
 	default n
 	help
 	  This enables thermal monitoring capability in the kernel in the
 	  absence of a system wide thermal monitoring entity. Monitors temperature
 	  via a specified ADC channel on the VADC device, and limits the max frequency
-	  of the CPU cores accordingly. The code is simple, allows for up to 8 throttling
+	  of the CPU cores accordingly. The code is simple, allows for multiple throttling
```

Comment:
The extra zones aren't really useful on the OPX. My last throttle zone already forces the maxfreq down to 729 MHz, so I don't think we need more throttling steps after that.